library/test/src/options.rs

```diff
@@ -38,6 +38,15 @@ pub enum OutputFormat {
     Junit,
 }
 
+impl OutputFormat {
+    pub(crate) fn is_programmatic(&self) -> bool {
+        match self {
+            Self::Pretty | Self::Terse => true,
+            Self::Json | Self::Junit => false,
```

Comment:
Are `true` and `false` flipped here? Or am I misunderstanding something?

Reply:
... sorry